INSTALL.md

```diff
@@ -13,15 +13,10 @@ depending on your operating system and what platform you want to develop for.
 
 ## 1: Getting the Source
 
-Mapbox GL Native uses git submodules, so you'll need to initialize and update
-them after cloning the source from Git.
+Clone the git repository:
 
     git clone https://github.com/mapbox/mapbox-gl-native.git
     cd mapbox-gl-native
-    git submodule init
-    git submodule update
```

Comment:
Submodules are still used for the KIF-based iOS SDK UI tests, so I moved this step to the iOS SDK testing guide in 102e62ae5582ec5cad5d8bbe3d42f99b875d143e. Once enough of the KIF tests have been ported to XCUITest in #4837, we can delete the KIF tests outright, and the two remaining submodules will be no more.